sdk/cosmosdb/cosmos/src/request/FeedOptions.ts

```diff
@@ -97,4 +96,15 @@ export interface FeedOptions extends SharedOptions, DedicatedGatewayRequestOptio
    * but you still want to restrict it to a single partition. Example: an end user specified query.
    */
   partitionKey?: any;
+  /**
+   * Sets the staleness value associated with the request in the Azure CosmosDB service. For requests where the {@link
+   * com.azure.cosmos.ConsistencyLevel} is {@link com.azure.cosmos.ConsistencyLevel#EVENTUAL}, responses from the
+   * integrated cache are guaranteed to be no staler than value indicated by this maxIntegratedCacheStaleness. When the
+   * consistency level is not set, this property is ignored. * Beta *
```

Comment:
I think that having this as a beta property this way is risky. The * beta * word at the end could easily be missed and our customers can start using a beta property unknowingly. 

If we merge this PR like this, we are making  `maxIntegratedCacheStalenessInMs` a stable property and committing to not breaking it. Any braking changes on this property will need a major version bump to release

I'd like to hear your toughts @bterlson, @xirzec , @witemple-msft

Reply:
Let's be more clear about this property being beta, what about something like this

```ts
 /**
   *  **(Beta - There may be breaking changes in this property in future releases)**
   * Sets the staleness value associated with the request in the Azure CosmosDB service. For requests where the {@link
   * com.azure.cosmos.ConsistencyLevel} is {@link com.azure.cosmos.ConsistencyLevel#EVENTUAL}, responses from the
   * integrated cache are guaranteed to be no staler than value indicated by this maxIntegratedCacheStaleness. When the
   * consistency level is not set, this property is ignored. 
   * */
```

Reply:
@kushagraThapar, will this work for our customers in the absence of a beta solution? I believe the property itself is stable because it already exists on the cosmos service, and it is currently not passed. Also, the service will not throw an error if this parameter is correct, and the service will ignore this parameter if it is incorrect or missing.